cypress/integration/registrar.spec.js

```diff
@@ -0,0 +1,30 @@
+const ROOT = Cypress.env('ROOT')
+const NAME_ROOT = Cypress.env('NAME_ROOT')
+
+describe('PermanentRegistrar', () => {
+  it('can visit a name and register it', () => {
+    cy.visit(`${ROOT}/name/vitalik.eth`)
+    cy.getByTestId('request-register-button').click()
```

Comment:
So is this relying on the fact that wait time is 1 sec  and the `click()` would wait for one sec? you may want to add that on the comment as it is not clear. Also this can be more explicit once we have a way to mine x block to time travel within the test (which probably needed for end of migration test as we discussed).

Reply:
It would just keep trying to click until it times out. There is no 1 second wait. But I am assuming instant or short blocktime